x-pack/legacy/plugins/siem/public/pages/case/components/all_cases/index.tsx

```diff
@@ -88,55 +122,141 @@ export const AllCases = React.memo(() => {
     [data, queryParams]
   );
 
+  const getBulkItemsPopoverContent = useCallback(
+    (closePopover: () => void) => (
+      <EuiContextMenuPanel
+        items={getBulkItems({
+          closePopover,
+          selectedCases,
+        })}
+      />
+    ),
+    [selectedCases]
+  );
+
   const sorting: EuiTableSortingType<Case> = {
     sort: { field: queryParams.sortField, direction: queryParams.sortOrder },
   };
+  const euiBasicTableSelectionProps = useMemo<EuiTableSelectionType<Case>>(
+    () => ({
+      selectable: (item: Case) => true,
+      onSelectionChange: setSelectedCases,
+    }),
+    [selectedCases]
+  );
+  const isCasesLoading = useMemo(
```

Comment:
removing the isLoading will simplify this logic, but we still need an `initLoading` state to show the initial loading design and then we can have the night rider's kibana loading so people can still play with the table

Reply:
![bug](https://user-images.githubusercontent.com/6935300/75937681-e6210e80-5e42-11ea-8568-e70dbfddbe92.gif)
got this solved but made a new bug, you can click close case again while the case is updating. ill need to add something to disable that row, because its fine if you click close case on the row below it

Reply:
ill fix in a follow up so you can get your PR merged tho @XavierM